packages/design-system/src/components/DateField/__snapshots__/DateField.test.jsx.snap

```diff
@@ -6,6 +6,8 @@ exports[`DateField has custom yearMax and yearMin 1`] = `
 >
   <legend
     className="ds-c-label"
+    component="legend"
```

Comment:
@line47 the addition of `component` and `hint` in the snapshots means that they are being passed to the html element inside `labelProps`, which we dont want.